bin/salita

```diff
@@ -13,6 +13,25 @@ var boolishToBool = function boolishToBool(value) {
   return value;
 };
 
+var hasDuplicates = function (arr) {
+  var len = arr.length - 1;
+  while (len >= 0) {
+    if (arr.indexOf(arr[len]) !== len) {
+      return true;
+    }
+    len -= 1;
+  }
+  return false;
+};
+
+var sections = ['dep', 'dev', 'peer', 'opt'];
```

Comment:
since "opt" isn't actually supported right now, it probably shouldn't be included right now?

Also, it'd probably be better if the main salita function exports this list rather than duplicating it in `bin/salita`